src/runtime/cli/config/configuration-qemu.toml.in

```diff
@@ -319,6 +319,34 @@ valid_file_mem_backends = @DEFVALIDFILEMEMBACKENDS@
 # Default 0-sized value means unlimited rate.
 #tx_rate_limiter_max_rate = 0
 
+# pvpanic device is a simulated ISA device, through which a guest panic
+# event is sent to qemu, and a QMP GUEST_PANICKED event is generated.
+# This allows management apps to be notified and respond to the event.
+#
+# Note:
+#   To enable pvpanic device, you must set
+#   CONFIG_PVPANIC=y for your guest kernel.
+#pvpanic_enabled=false
```

Comment:
fwics, with this PR we have the following behaviour:

# Summary

| `pvpanic_enabled` | `guest_memory_dump_path` | Outcome |
|-|-|-|
| `false` | "" | NOP |
| `true` | "" | NOP (*) |
| `false` | valid path | NOP |
| `true` | valid path | Dump guest memory and metadata |

Key:

(*) - currently does nothing, but ideally would pass the event details to some sort of handler to allow the admin to know a panic occurred.

# Conclusion

Assuming this summary is correct, I think you can remove the `pvpanic_enabled=` option for now. When we have a plan for the panic handler (*), we can add that option to the config.

Reply:
Agree, I think we can get away with using just the `guest_memory_dump_path` for now.

Reply:
Ok, I'll delete `pvpanic_enabled` for now